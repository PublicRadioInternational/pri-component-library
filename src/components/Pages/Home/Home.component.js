/**
 * @file Home.component.js
 * Exports the homepage.
 */

import React from 'react';

import Header from '../../Organisms/Header/Header.component';
import Layout from '../../Organisms/Main/Layout.component';
import Footer from '../../Organisms/Footer/Footer.component';

/**
 * Component that renders the homepage.
 */
const Home = () => (
  <div>
    <Header />
    <Layout />
    <Footer />
  </div>
);

export default Home;
