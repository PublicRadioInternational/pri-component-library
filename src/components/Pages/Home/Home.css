/* import colors... */
@value colors: "../../00_global/colors.css";
@value grayLighter, grayDark from colors;
/* import fonts... */
@value fonts: "../../00_global/fonts.css";
@value copy from fonts;

.outerWrap {
  background: grayLighter;
  color: grayDark;
  font-family: copy;
}

.drawerBg {
  background: rgba(0, 0, 0, 0.5);
  bottom: 0;
  top: 0;
  left: 0;
  position: fixed;
  width: 100%;
  height: 100%;
  z-index: 4000;
  opacity: 0.001;
  transform: translateZ(0);
  visibility: hidden;
}
