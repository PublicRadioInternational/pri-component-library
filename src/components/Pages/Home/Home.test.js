/**
 * @file Button.test.js
 * Contains tests for Button.component.js.
 */

import React from 'react';
import renderer from 'react-test-renderer';

import Home from './Home.component';

describe('<Home />', () => {
  it('Matches the Header snapshot', () => {
    const component = renderer.create(<Home />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
