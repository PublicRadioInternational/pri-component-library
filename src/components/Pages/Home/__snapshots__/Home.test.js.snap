// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Matches the Header snapshot 1`] = `
<div
  className="outerWrap"
>
  <header
    className="siteHeader"
  >
    <a
      className="logo"
      href="/"
    >
      <svg
        aria-hidden={null}
        className="disableHover"
        fill="currentcolor"
        viewBox={undefined}
      >
        <title>
          PRI
        </title>
        <use
          xlinkHref="#undefined"
        />
      </svg>
    </a>
    <div
      className="buttons"
    >
      <div
        className="dropdownGrp button"
      >
        <a
          className="btnMobileWhite btnGrp"
          href="https://www.pri.org/listen"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="inlineSvg null"
            fill="currentcolor"
            viewBox={undefined}
          >
            <use
              xlinkHref="#undefined"
            />
          </svg>
          <span
            className="textLabel textLabelMobile"
          >
            Listen
          </span>
        </a>
        <button
          aria-expanded={null}
          aria-haspopup={true}
          aria-label="open menu"
          className="btnMobileWhite btnDropdownWhite btnHide"
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="dropdownGrp button"
      >
        <a
          className="btnMobileWhite btnGrp"
          href="https://www.pri.org/newsletters"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="inlineSvg null"
            fill="currentcolor"
            viewBox={undefined}
          >
            <use
              xlinkHref="#undefined"
            />
          </svg>
          <span
            className="textLabel textLabelMobile"
          >
            Newsletters
          </span>
        </a>
        <button
          aria-expanded={null}
          aria-haspopup={true}
          aria-label="open menu"
          className="btnMobileWhite btnDropdownWhite btnHide"
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="dropdownGrp button"
      >
        <a
          className="btnMobileWhite btnGrp"
          href="https://interactive-dev.pri.org/staging/prototypes/homepage/iteration-2.html#"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="inlineSvg null"
            fill="currentcolor"
            viewBox={undefined}
          >
            <use
              xlinkHref="#undefined"
            />
          </svg>
          <span
            className="textLabel textLabelMobile"
          >
            Search
          </span>
        </a>
        <button
          aria-expanded={null}
          aria-haspopup={true}
          aria-label="open menu"
          className="btnMobileWhite btnDropdownWhite btnHide"
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="dropdownGrp button"
      >
        <a
          className="btnMobileOrange btnGrp"
          href="https://interactive-dev.pri.org/staging/prototypes/homepage/iteration-2.html#"
          onClick={[Function]}
        >
          <svg
            aria-hidden={true}
            className="inlineSvg null"
            fill="currentcolor"
            viewBox={undefined}
          >
            <use
              xlinkHref="#undefined"
            />
          </svg>
          <span
            className="textLabel textLabelMobile"
          >
            Donate
          </span>
        </a>
        <button
          aria-expanded={null}
          aria-haspopup={true}
          aria-label="open menu"
          className="btnMobileOrange btnDropdownOrange btnHide"
          onClick={[Function]}
          type="button"
        />
      </div>
    </div>
  </header>
  <div
    className="drawerBg"
  />
</div>
`;
